protos/foundation/protos/doc/entity.proto

```diff
@@ -2,11 +2,13 @@ syntax = "proto2";
 
 package doc;
 
+import "foundation/protos/geometry.proto";
 import "foundation/protos/ocr/types.proto";
 
+/* Union type holds one of these Entity variants. */
 message Entity {
 	oneof payload {
-		OcrWord ocr_word = 1;
+		Word word = 1;
```

Comment:
I would use "DollarValue", "GBPValue", etc., instead of "currency". What is the trade-off from the user's perspective?